lib/assets/stylesheets/bootstrap-without-forms.css.scss

```diff
@@ -1,53 +1,14 @@
7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{background-color:#f2dede;border-color:#ebccd1;color:#a94442}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{overflow:hidden;height:20px;margin-bottom:20px;background-color:#f5f5f5;border-radius:4px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,0.1);box-shadow:inset 0 1px 2px rgba(0,0,0,0.1)}.progress-bar{float:left;width:0%;height:100%;font-size:12px;line-height:20px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,0.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,0.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-striped .progress-bar,.progress-bar-striped{background-image:-webkit-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:-o-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);-webkit-background-size:40px 40px;background-size:40px 40px}.progress.active .progress-bar,.progress-bar.active{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:-o-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:-o-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%,
```

Comment:
Color literals like `#ff0` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#c0c0c0` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#999` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#337ab7` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#23527c` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#777` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#fcf8e3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#286090` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#3c763d` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#2b542c` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#31708f` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#245269` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#8a6d3b` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#66512c` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#a94442` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#843534` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#dff0d8` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#c1e2b3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#d9edf7` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#afd9ee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#f7ecb5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#f2dede` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#e4b9b9` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#c7254e` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#f9f2f4` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.25)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#f5f5f5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#f9f9f9` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#e8e8e8` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#d0e9c6` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#c4e3f3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#faf2cc` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ebcccc` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.125)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#e6e6e6` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#8c8c8c` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#adadad` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#d4d4d4` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#2e6da4` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#122b40` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#204d74` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#5cb85c` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#4cae4c` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#449d44` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#255625` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#398439` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#5bc0de` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#46b8da` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#31b0d5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#1b6d85` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#269abc` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#f0ad4e` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#eea236` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ec971f` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#985f0d` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#d58512` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#d9534f` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#d43f3a` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#c9302c` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#761c19` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ac2925` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.15)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.175)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#e5e5e5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#262626` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#555` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#5e5e5e` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#d5d5d5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#d6e9c6` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#c9e2b3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#bce8f1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#a6e1ec` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#faebcc` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#f7e1b5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ebccd1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#e4b9c0` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.1)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(255, 255, 255, 0.15)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#c7ddef` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.05)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#e3e3e3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.2)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.5)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#f7f7f7` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ebebeb` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.6)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.0001)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Use `//` comments everywhere<br>Property `outline` already defined on line 14<br>Property `border-top` already defined on line 14<br>Property `border` already defined on line 14<br>Property `border-bottom` already defined on line 14<br>Property `border-color` already defined on line 14<br>Property `text-align` already defined on line 14<br>Property `border-top-color` already defined on line 14<br>Property `border-right-color` already defined on line 14<br>Property `border-bottom-color` already defined on line 14<br>Property `border-left-color` already defined on line 14<br>Property `background-color` already defined on line 14<br>Files should end with a trailing newline<br>!important should not be used<br>`.2` should be written with a leading zero as `0.2`<br>`.65` should be written with a leading zero as `0.65`<br>`.15` should be written with a leading zero as `0.15`<br>`.35` should be written with a leading zero as `0.35`<br>`.6` should be written with a leading zero as `0.6`<br>`.3` should be written with a leading zero as `0.3`<br>`.25` should be written with a leading zero as `0.25`<br>`.5` should be written with a leading zero as `0.5`<br>`.9` should be written with a leading zero as `0.9`<br>Merge rule `html` with rule on line 14<br>Merge rule `body` with rule on line 14<br>Merge rule `a` with rule on line 14<br>Merge rule `figure` with rule on line 14<br>Merge rule `img` with rule on line 14<br>Merge rule `hr` with rule on line 14<br>Merge rule `code,kbd,pre,samp` with rule on line 14<br>Merge rule `kbd kbd` with rule on line 14<br>Merge rule `pre` with rule on line 14<br>Merge rule `pre code` with rule on line 14<br>Merge rule `table` with rule on line 14<br>Merge rule `.table .table` with rule on line 14<br>Merge rule `table col[class*="col-"]` with rule on line 14<br>Merge rule `.btn-default:active,.btn-default.active,.open>.dropdown-toggle.btn-default` with rule on line 14<br>Merge rule `.btn-default .badge` with rule on line 14<br>Merge rule `.btn-primary:active,.btn-primary.active,.open>.dropdown-toggle.btn-primary` with rule on line 14<br>Merge rule `.btn-primary .badge` with rule on line 14<br>Merge rule `.btn-success:active,.btn-success.active,.open>.dropdown-toggle.btn-success` with rule on line 14<br>Merge rule `.btn-success .badge` with rule on line 14<br>Merge rule `.btn-info:active,.btn-info.active,.open>.dropdown-toggle.btn-info` with rule on line 14<br>Merge rule `.btn-info .badge` with rule on line 14<br>Merge rule `.btn-warning:active,.btn-warning.active,.open>.dropdown-toggle.btn-warning` with rule on line 14<br>Merge rule `.btn-warning .badge` with rule on line 14<br>Merge rule `.btn-danger:active,.btn-danger.active,.open>.dropdown-toggle.btn-danger` with rule on line 14<br>Merge rule `.btn-danger .badge` with rule on line 14<br>Merge rule `.fade.in` with rule on line 14<br>Merge rule `.collapse.in` with rule on line 14<br>Merge rule `.dropdown-menu.pull-right` with rule on line 14<br>Merge rule `.dropdown-menu .divider` with rule on line 14<br>Merge rule `.btn-group.open .dropdown-toggle.btn-link` with rule on line 14<br>Merge rule `.btn .caret` with rule on line 14<br>Merge rule `.nav>li.disabled>a` with rule on line 14<br>Merge rule `.nav .nav-divider` with rule on line 14<br>Merge rule `.nav-tabs.nav-justified` with rule on line 14<br>Merge rule `.nav-tabs.nav-justified>li` with rule on line 14<br>Merge rule `.nav-tabs.nav-justified>li>a` with rule on line 14<br>Merge rule `.nav-tabs.nav-justified>.dropdown .dropdown-menu` with rule on line 14<br>Merge rule `.nav-tabs .dropdown-menu` with rule on line 14<br>Merge rule `.pager li` with rule on line 14<br>Merge rule `.btn .label` with rule on line 14<br>Merge rule `.btn .badge` with rule on line 14<br>Merge rule `.jumbotron p` with rule on line 14<br>Merge rule `.jumbotron .container` with rule on line 14<br>Merge rule `.thumbnail .caption` with rule on line 14<br>Merge rule `.alert h4` with rule on line 14<br>Merge rule `.alert .alert-link` with rule on line 14<br>Merge rule `.alert-success hr` with rule on line 14<br>Merge rule `.alert-success .alert-link` with rule on line 14<br>Merge rule `.alert-info hr` with rule on line 14<br>Merge rule `.alert-info .alert-link` with rule on line 14<br>Merge rule `.alert-warning hr` with rule on line 14<br>Merge rule `.alert-warning .alert-link` with rule on line 14<br>Merge rule `.alert-danger hr` with rule on line 14<br>Merge rule `.alert-danger .alert-link` with rule on line 14<br>Merge rule `.media-object.img-thumbnail` with rule on line 14<br>Merge rule `button.list-group-item` with rule on line 14<br>Merge rule `.panel-group .panel` with rule on line 14<br>Merge rule `.panel-group .panel+.panel` with rule on line 14<br>Merge rule `.panel-group .panel-heading` with rule on line 14<br>Merge rule `.panel-group .panel-footer` with rule on line 14<br>Merge rule `.panel-group .panel-footer+.panel-collapse .panel-body` with rule on line 14<br>Merge rule `.panel-default>.panel-heading .badge` with rule on line 14<br>Merge rule `.panel-primary>.panel-heading .badge` with rule on line 14<br>Merge rule `.panel-success>.panel-heading .badge` with rule on line 14<br>Merge rule `.panel-info>.panel-heading .badge` with rule on line 14<br>Merge rule `.panel-warning>.panel-heading .badge` with rule on line 14<br>Merge rule `.panel-danger>.panel-heading .badge` with rule on line 14<br>Merge rule `.well blockquote` with rule on line 14<br>Merge rule `button.close` with rule on line 14<br>Merge rule `.modal.fade .modal-dialog` with rule on line 14<br>Merge rule `.modal.in .modal-dialog` with rule on line 14<br>Merge rule `.modal-open .modal` with rule on line 14<br>Merge rule `.modal-backdrop.fade` with rule on line 14<br>Merge rule `.modal-backdrop.in` with rule on line 14<br>Merge rule `.modal-header .close` with rule on line 14<br>Merge rule `.modal-footer .btn+.btn` with rule on line 14<br>Merge rule `.modal-footer .btn-group .btn+.btn` with rule on line 14<br>Merge rule `.modal-footer .btn-block+.btn-block` with rule on line 14<br>Merge rule `.tooltip.in` with rule on line 14<br>Merge rule `.tooltip.top` with rule on line 14<br>Merge rule `.tooltip.right` with rule on line 14<br>Merge rule `.tooltip.bottom` with rule on line 14<br>Merge rule `.tooltip.left` with rule on line 14<br>Merge rule `.tooltip.top .tooltip-arrow` with rule on line 14<br>Merge rule `.tooltip.top-left .tooltip-arrow` with rule on line 14<br>Merge rule `.tooltip.top-right .tooltip-arrow` with rule on line 14<br>Merge rule `.tooltip.right .tooltip-arrow` with rule on line 14<br>Merge rule `.tooltip.left .tooltip-arrow` with rule on line 14<br>Merge rule `.tooltip.bottom .tooltip-arrow` with rule on line 14<br>Merge rule `.tooltip.bottom-left .tooltip-arrow` with rule on line 14<br>Merge rule `.tooltip.bottom-right .tooltip-arrow` with rule on line 14<br>Merge rule `.popover.top` with rule on line 14<br>Merge rule `.popover.right` with rule on line 14<br>Merge rule `.popover.bottom` with rule on line 14<br>Merge rule `.popover.left` with rule on line 14<br>Merge rule `.popover.top>.arrow` with rule on line 14<br>Merge rule `.popover.top>.arrow:after` with rule on line 14<br>Merge rule `.popover.right>.arrow` with rule on line 14<br>Merge rule `.popover.right>.arrow:after` with rule on line 14<br>Merge rule `.popover.bottom>.arrow` with rule on line 14<br>Merge rule `.popover.bottom>.arrow:after` with rule on line 14<br>Merge rule `.popover.left>.arrow` with rule on line 14<br>Merge rule `.popover.left>.arrow:after` with rule on line 14<br>Merge rule `.carousel-inner>.active.left` with rule on line 14<br>Merge rule `.carousel-inner>.active.right` with rule on line 14<br>Merge rule `.carousel-control.left` with rule on line 14<br>Merge rule `.carousel-control.right` with rule on line 14<br>Merge rule `.carousel-control .icon-prev:before` with rule on line 14<br>Merge rule `.carousel-control .icon-next:before` with rule on line 14<br>Merge rule `.carousel-indicators li` with rule on line 14<br>Merge rule `.carousel-indicators .active` with rule on line 14<br>Merge rule `.carousel-caption .btn` with rule on line 14<br>Rule set contains (11/10) properties<br>Rule set contains (19/10) properties<br>Rule set contains (12/10) properties<br>Rule set contains (20/10) properties<br>Rule set contains (13/10) properties<br>Rule set contains (21/10) properties<br>Rule set contains (31/10) properties<br>Properties should be ordered -moz-box-sizing, -webkit-box-sizing, box-sizing, height<br>Properties should be ordered -moz-box-sizing, -webkit-box-sizing, box-sizing, padding<br>Properties should be ordered -webkit-appearance, -moz-box-sizing, -webkit-box-sizing, box-sizing<br>Properties should be ordered background, -webkit-box-shadow, box-shadow, color, text-shadow<br>Properties should be ordered display, font-family, -moz-osx-font-smoothing, -webkit-font-smoothing, font-style, font-weight, line-height, position, top<br>Properties should be ordered -moz-box-sizing, -webkit-box-sizing, box-sizing<br>Properties should be ordered background-color, color, font-family, font-size, line-height<br>Properties should be ordered display, height, max-width<br>Properties should be ordered background-color, border, border-radius, display, height, line-height, max-width, padding, -o-transition, -webkit-transition, transition<br>Properties should be ordered border, border-top, margin-bottom, margin-top<br>Properties should be ordered border, clip, height, margin, overflow, padding, position, width<br>Properties should be ordered clip, height, margin, overflow, position, width<br>Properties should be ordered color, font-family, font-weight, line-height<br>Properties should be ordered color, font-weight, line-height<br>Properties should be ordered margin-bottom, margin-top<br>Properties should be ordered font-size, font-weight, line-height, margin-bottom<br>Properties should be ordered background-color, color<br>Properties should be ordered border-bottom, margin, padding-bottom<br>Properties should be ordered list-style, padding-left<br>Properties should be ordered list-style, margin-left, padding-left<br>Properties should be ordered clear, float, overflow, text-align, text-overflow, white-space, width<br>Properties should be ordered border-bottom, cursor<br>Properties should be ordered border-left, font-size, margin, padding<br>Properties should be ordered color, display, font-size, line-height<br>Properties should be ordered border-left, border-right, padding-left, padding-right, text-align<br>Properties should be ordered font-style, line-height, margin-bottom<br>Properties should be ordered background-color, border-radius, color, font-size, padding<br>Properties should be ordered background-color, border-radius, -webkit-box-shadow, box-shadow, color, font-size, padding<br>Properties should be ordered -webkit-box-shadow, box-shadow, font-size, font-weight, padding<br>Properties should be ordered background-color, border, border-radius, color, display, font-size, line-height, margin, padding, word-break, word-wrap<br>Properties should be ordered background-color, border-radius, color, font-size, padding, white-space<br>Properties should be ordered margin-left, margin-right, padding-left, padding-right<br>Properties should be ordered min-height, padding-left, padding-right, position<br>Properties should be ordered color, padding-bottom, padding-top, text-align<br>Properties should be ordered margin-bottom, max-width, width<br>Properties should be ordered border-top, line-height, padding, vertical-align<br>Properties should be ordered border-bottom, vertical-align<br>Properties should be ordered display, float, position<br>Properties should be ordered min-height, overflow-x<br>Properties should be ordered border, margin-bottom, -ms-overflow-style, overflow-y, width<br>Properties should be ordered background-image, border, border-radius, cursor, display, font-size, font-weight, line-height, margin-bottom, padding, text-align, -ms-touch-action, touch-action, -moz-user-select, -ms-user-select, -webkit-user-select, user-select, vertical-align, white-space<br>Properties should be ordered background-image, -webkit-box-shadow, box-shadow, outline<br>Properties should be ordered -webkit-box-shadow, box-shadow, cursor, filter, opacity<br>Properties should be ordered background-color, border-color, color<br>Properties should be ordered border-radius, color, font-weight<br>Properties should be ordered background-color, color, text-decoration<br>Properties should be ordered border-radius, font-size, line-height, padding<br>Properties should be ordered opacity, -o-transition, -webkit-transition, transition<br>Properties should be ordered height, overflow, position, -o-transition-duration, -webkit-transition-duration, transition-duration, -o-transition-property, -webkit-transition-property, transition-property, -o-transition-timing-function, -webkit-transition-timing-function, transition-timing-function<br>Properties should be ordered border-left, border-right, border-top, border-top, display, height, margin-left, vertical-align, width<br>Properties should be ordered -webkit-background-clip, background-clip, background-color, border, border, border-radius, -webkit-box-shadow, box-shadow, display, float, font-size, left, list-style, margin, min-width, padding, position, text-align, top, z-index<br>Properties should be ordered left, right<br>Properties should be ordered background-color, height, margin, overflow<br>Properties should be ordered clear, color, display, font-weight, line-height, padding, white-space<br>Properties should be ordered background-color, color, outline, text-decoration<br>Properties should be ordered background-color, background-image, cursor, filter, text-decoration<br>Properties should be ordered color, display, font-size, line-height, padding, white-space<br>Properties should be ordered bottom, left, position, right, top, z-index<br>Properties should be ordered border-bottom, border-bottom, border-top, content<br>Properties should be ordered bottom, margin-bottom, top<br>Properties should be ordered display, position, vertical-align<br>Properties should be ordered float, position<br>Properties should be ordered border-bottom-width, border-width<br>Properties should be ordered display, float, max-width, width<br>Properties should be ordered margin-left, margin-top<br>Properties should be ordered border-bottom-left-radius, border-bottom-right-radius, border-top-left-radius, border-top-right-radius<br>Properties should be ordered border-bottom-left-radius, border-bottom-right-radius<br>Properties should be ordered border-top-left-radius, border-top-right-radius<br>Properties should be ordered border-collapse, display, table-layout, width<br>Properties should be ordered display, float, width<br>Properties should be ordered clip, pointer-events, position<br>Properties should be ordered list-style, margin-bottom, padding-left<br>Properties should be ordered display, position<br>Properties should be ordered display, padding, position<br>Properties should be ordered background-color, text-decoration<br>Properties should be ordered background-color, color, cursor, text-decoration<br>Properties should be ordered border, border-radius, line-height, margin-right<br>Properties should be ordered background-color, border, border-bottom-color, color, cursor<br>Properties should be ordered border-bottom, width<br>Properties should be ordered margin-bottom, text-align<br>Properties should be ordered left, top<br>Properties should be ordered border-radius, margin-right<br>Properties should be ordered border-top-left-radius, border-top-right-radius, margin-top<br>Properties should be ordered background-color, border-radius, list-style, margin-bottom, padding<br>Properties should be ordered color, content, padding<br>Properties should be ordered border-radius, display, margin, padding-left<br>Properties should be ordered background-color, border, color, float, line-height, margin-left, padding, position, text-decoration<br>Properties should be ordered border-bottom-left-radius, border-top-left-radius, margin-left<br>Properties should be ordered background-color, border-color, color, z-index<br>Properties should be ordered background-color, border-color, color, cursor, z-index<br>Properties should be ordered background-color, border-color, color, cursor<br>Properties should be ordered font-size, line-height, padding<br>Properties should be ordered list-style, margin, padding-left, text-align<br>Properties should be ordered background-color, border, border-radius, display, padding<br>Properties should be ordered background-color, color, cursor<br>Properties should be ordered border-radius, color, display, font-size, font-weight, line-height, padding, text-align, vertical-align, white-space<br>Properties should be ordered color, cursor, text-decoration<br>Properties should be ordered background-color, border-radius, color, display, font-size, font-weight, line-height, min-width, padding, text-align, vertical-align, white-space<br>Properties should be ordered padding, top<br>Properties should be ordered background-color, color, margin-bottom, padding-bottom, padding-top<br>Properties should be ordered font-size, font-weight, margin-bottom<br>Properties should be ordered padding-bottom, padding-top<br>Properties should be ordered background-color, border, border-radius, display, line-height, margin-bottom, padding, -o-transition, -webkit-transition, transition<br>Properties should be ordered color, padding<br>Properties should be ordered border, border-radius, margin-bottom, padding<br>Properties should be ordered color, margin-top<br>Properties should be ordered color, position, right, top<br>Properties should be ordered background-color, border-radius, -webkit-box-shadow, box-shadow, height, margin-bottom, overflow<br>Properties should be ordered background-color, -webkit-box-shadow, box-shadow, color, float, font-size, height, line-height, text-align, -o-transition, -webkit-transition, transition, width<br>Properties should be ordered -o-animation, -webkit-animation, animation<br>Properties should be ordered overflow, zoom<br>Properties should be ordered background-color, border, display, margin-bottom, padding, position<br>Properties should be ordered border-bottom-left-radius, border-bottom-right-radius, margin-bottom<br>Properties should be ordered text-align, width<br>Properties should be ordered line-height, margin-bottom<br>Properties should be ordered background-color, border, border-radius, -webkit-box-shadow, box-shadow, margin-bottom<br>Properties should be ordered border-bottom, border-top-left-radius, border-top-right-radius, padding<br>Properties should be ordered color, font-size, margin-bottom, margin-top<br>Properties should be ordered background-color, border-bottom-left-radius, border-bottom-right-radius, border-top, padding<br>Properties should be ordered border-radius, border-width<br>Properties should be ordered border-top, border-top-left-radius, border-top-right-radius<br>Properties should be ordered border-bottom, border-bottom-left-radius, border-bottom-right-radius<br>Properties should be ordered border-radius, margin-bottom<br>Properties should be ordered display, height, overflow, padding, position<br>Properties should be ordered border, bottom, height, left, position, top, width<br>Properties should be ordered background-color, border, border-radius, -webkit-box-shadow, box-shadow, margin-bottom, min-height, padding<br>Properties should be ordered border-radius, padding<br>Properties should be ordered color, filter, float, font-size, font-weight, line-height, opacity, text-shadow<br>Properties should be ordered color, cursor, filter, opacity, text-decoration<br>Properties should be ordered -webkit-appearance, background, border, cursor, padding<br>Properties should be ordered bottom, display, left, outline, overflow, -webkit-overflow-scrolling, position, right, top, z-index<br>Properties should be ordered -ms-transform, -o-transform, -webkit-transform, transform, -o-transition, -webkit-transition, transition<br>Properties should be ordered -ms-transform, -o-transform, -webkit-transform, transform<br>Properties should be ordered margin, position, width<br>Properties should be ordered -webkit-background-clip, background-clip, background-color, border, border, border-radius, -webkit-box-shadow, box-shadow, outline, position<br>Properties should be ordered background-color, bottom, left, position, right, top, z-index<br>Properties should be ordered filter, opacity<br>Properties should be ordered border-bottom, padding<br>Properties should be ordered line-height, margin<br>Properties should be ordered padding, position<br>Properties should be ordered border-top, padding, text-align<br>Properties should be ordered margin-bottom, margin-left<br>Properties should be ordered height, overflow, position, top, width<br>Properties should be ordered margin, width<br>Properties should be ordered display, filter, font-family, font-size, font-style, font-weight, letter-spacing, line-break, line-height, opacity, position, text-align, text-align, text-decoration, text-shadow, text-transform, white-space, word-break, word-spacing, word-wrap, z-index<br>Properties should be ordered background-color, border-radius, color, max-width, padding, text-align<br>Properties should be ordered border-color, border-style, height, position, width<br>Properties should be ordered border-top-color, border-width, bottom, left, margin-left<br>Properties should be ordered border-top-color, border-width, bottom, margin-bottom, right<br>Properties should be ordered border-top-color, border-width, bottom, left, margin-bottom<br>Properties should be ordered border-right-color, border-width, left, margin-top, top<br>Properties should be ordered border-left-color, border-width, margin-top, right, top<br>Properties should be ordered border-bottom-color, border-width, left, margin-left, top<br>Properties should be ordered border-bottom-color, border-width, margin-top, right, top<br>Properties should be ordered border-bottom-color, border-width, left, margin-top, top<br>Properties should be ordered -webkit-background-clip, background-clip, background-color, border, border, border-radius, -webkit-box-shadow, box-shadow, display, font-family, font-size, font-style, font-weight, left, letter-spacing, line-break, line-height, max-width, padding, position, text-align, text-align, text-decoration, text-shadow, text-transform, top, white-space, word-break, word-spacing, word-wrap, z-index<br>Properties should be ordered background-color, border-bottom, border-radius, font-size, margin, padding<br>Properties should be ordered border-color, border-style, display, height, position, width<br>Properties should be ordered border-bottom-width, border-top-color, border-top-color, bottom, left, margin-left<br>Properties should be ordered border-bottom-width, border-top-color, bottom, content, margin-left<br>Properties should be ordered border-left-width, border-right-color, border-right-color, left, margin-top, top<br>Properties should be ordered border-left-width, border-right-color, bottom, content, left<br>Properties should be ordered border-bottom-color, border-bottom-color, border-top-width, left, margin-left, top<br>Properties should be ordered border-bottom-color, border-top-width, content, margin-left, top<br>Properties should be ordered border-left-color, border-left-color, border-right-width, margin-top, right, top<br>Properties should be ordered border-left-color, border-right-width, bottom, content, right<br>Properties should be ordered overflow, position, width<br>Properties should be ordered display, position, -o-transition, -webkit-transition, transition<br>Properties should be ordered -webkit-backface-visibility, backface-visibility, -webkit-perspective, perspective, -o-transition, -webkit-transition, transition<br>Properties should be ordered left, -webkit-transform, transform<br>Properties should be ordered background-color, bottom, color, filter, font-size, left, opacity, position, text-align, text-shadow, top, width<br>Properties should be ordered background-image, background-image, background-image, background-image, background-repeat, filter, left, right<br>Properties should be ordered color, filter, opacity, outline, text-decoration<br>Properties should be ordered display, margin-top, position, top, z-index<br>Properties should be ordered margin-right, right<br>Properties should be ordered font-family, height, line-height, width<br>Properties should be ordered bottom, left, list-style, margin-left, padding-left, position, text-align, width, z-index<br>Properties should be ordered background-color, background-color, border, border-radius, cursor, display, height, margin, text-indent, width<br>Properties should be ordered background-color, height, margin, width<br>Properties should be ordered bottom, color, left, padding-bottom, padding-top, position, right, text-align, text-shadow, z-index<br>Properties should be ordered font-size, height, margin-top, width<br>Properties should be ordered left, padding-bottom, right<br>Properties should be ordered background-color, border, color, font, text-shadow<br>Begin pseudo classes with a single colon: `:`<br>Begin pseudo elements with double colons: `::`<br>Avoid qualifying attribute selectors with an element.<br>Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector should have depth of applicability no greater than 2, but was 4<br>Selector should have depth of applicability no greater than 2, but was 5<br>Selector should have depth of applicability no greater than 2, but was 6<br>Each selector in a comma sequence should be on its own single line<br>Commas in lists should be followed by a single space<br>Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space<br>`opacity=65` should be written with a single space on each side of the operator: `opacity = 65`<br>`opacity=20` should be written with a single space on each side of the operator: `opacity = 20`<br>`opacity=50` should be written with a single space on each side of the operator: `opacity = 50`<br>`opacity=0` should be written with a single space on each side of the operator: `opacity = 0`<br>`opacity=90` should be written with a single space on each side of the operator: `opacity = 90`<br>Opening curly brace `{` should be preceded by one space<br>Prefer double-quoted strings<br>Declaration should be terminated by a semicolon<br>Avoid vendor prefixes.